include/boost/json/value.hpp

```diff
@@ -2479,12 +2480,13 @@ class value
             ! std::is_same<T, bool>::value,
                 T>::type
     {
+        using limits = std::numeric_limits<T>;
```

Comment:
Is the alias necessary for fixing the warnings?

Reply:
If you are going to change it use `DBL_MIN` and `DBL_MAX` or whatever those macros are called

Reply:
The alias isn't necessary, I just added it for convenience. We can use the macros instead